pycodestyle.py

```diff
@@ -2280,6 +2281,9 @@ def process_options(arglist=None, parse_argv=False, config_file=None,
     (options, args) = parser.parse_args(arglist)
     options.reporter = None
 
+    if verbose:  # If specified verbose, continue on verbosity
```

Comment:
I don't think this is necessary, nor do I think the addition to the signature on 2260 is necessary I'll explain in a different comment. This, however, does not help someone who is trying to pass `verbose=False` directly meaning that this feature is a bit half-baked.

Reply:
Would you suggest alternatively having the keyword argument default to `verbose=None`, and just check like this?
``` python3
if verbose is not None:
    options.verbose = verbose
```